programs/survival/sleep_ignore_fake_players.sc

```diff
@@ -17,7 +17,7 @@ in_dimension('overworld',
             global_allPlayerSleep = false;
             l = day_time() + 24000;
             day_time(l - l % 24000);
-            run('weather clear');
+            if(weather('rain')>0, weather('clear', round(rand(0x7FFFFFFF))))
```

Comment:
```suggestion
            // doing this also sets rain and thunder to 0 because of scarpet impl, exactly what vanilla does when sleeping (ServerWorld in yarn)
            if (weather('rain') > 0, weather('clear', weather('clear')))
```

Reply:
Yep running `weather('clear')` just before this line returns `0`.